inc/saiport.h

```diff
@@ -3133,6 +3133,9 @@ typedef enum _sai_port_stat_t
     /** Port stat out drop reasons range end */
     SAI_PORT_STAT_OUT_DROP_REASON_RANGE_END = 0x00002fff,
 
+    /** Port stat range end */
+    SAI_PORT_STAT_END = SAI_PORT_STAT_OUT_DROP_REASON_RANGE_END
```

Comment:
this kind of enum is not supported, can you use sai metadata generate to get all enums ?

Reply:
i see. then I can use SAI_PORT_STAT_OUT_DROP_REASON_RANGE_END directly as the extension start. is it ok?

Reply:
updated now.